### ISSUE TITLE
[InteractiveSegmentation] Add option to save Segmentation as Dicom

### ISSUE THREAD

**Issue by ashishsingh18** [CBICA/CaPTk2.0#141]
This will only be enabled if the input was also a Dicom. For Nifti this would simply be unavailable.

**Comment by dboun** [comment]
Nice catch!

If you notice, assuming your input image is a dicom:

- The seeds can be saved as a dicom
- The output segmentation can't.

So I'm assuming it is properties-related, and I should just copy them

**Comment by ashishsingh18** [comment]
I will need to see this working. This will be a much bigger fix as the 'save' doesn't give an option to save as dicom. 

1) We will need to add this on the UI. We can start with a button right under the 'Run' that will get enabled if the input was a dicom and if the segmentation completed successfully.

2) Then we will need to hook it to apt dicom writer with all the meta data from the input dicom.

**Comment by dboun** [comment]
Have you seen my PR on this? I think you are overthinking this.

The save context menu option *does* give an option to save as dicom. To test it, load a dicom image, click create seeds and try to save the seeds

**Comment by dboun** [comment]
I should emphasize that in the above it is a requirement that the loaded image is a dicom, eitherwise it is not an option

**Comment by ashishsingh18** [comment]
> 
> 
> Have you seen my PR on this? I think you are overthinking this.
> 
> The save context menu option _does_ give an option to save as dicom. To test it, load a dicom image, click create seeds and try to save the seeds

Right you are! I was looking at the incorrect place on older version (2016). Apparently dicom save option is available only when you have a segmentation. If you just load some data and try to save it, there is no option to save as dicom(which makes sense).

**Comment by ashishsingh18** [comment]
> 
> 
> I should emphasize that in the above it is a requirement that the loaded image is a dicom, eitherwise it is not an option

Yes the loaded image should be a dicom. Have you tried it btw with some dicom data?

**Comment by dboun** [comment]
> Have you tried it btw with some dicom data?

Yes! Before PR #144, only the seeds image could be saved as DICOM. After PR #144 both seeds and output segmentation can!
